wordpress-mobile/AztecEditor-iOS
# Fixes some issues when pasting content from Google Docs.

### Description:

Fixes #1056.

Adds support for CSS content that's bold, italic or underlined.  This also fixes pasting content with those styles from some 3rd party apps, such as Google Docs.

### Discussion:

Considering the complexity of this PR, please feel free to ask as many questions as necessary, or request improvements and changes.

This PR was posted as soon as I got it working so we can discuss the approach and adjust as necessary.

**Please note that:** I'll be adding unit tests and documentation before this PR is merged.

### Testing:

1. Launch Google Docs.
2. Add content to the Google doc containing any of the styles mentioned above.
3. Copy the content.
4. Paste the content in to Aztec.  Make sure it's properly rendered.
5. Switch to HTML mode and make sure the relevant CSS attributes are there.
6. Make sure Aztec hasn't added HTML elements for those styles.
7. Remove those styles from visual mode.
8. Switch back to HTML mode and make sure the CSS styles are gone.
